package.json

```diff
@@ -27,24 +27,24 @@
     "import:data:sample": "yarn start:dbs && yarn exec:local rm-data.sh || true && yarn setup:sample-data && cd ./packages/migrations && yarn setup:development",
     "import:data:sql": "yarn remote:import:data:sql -s local && yarn build:index:specimen",
     "import:data:zip": "yarn start:dbs && node ./packages/scripts/src/js/unpackDataFiles.js -s local && cd ./packages/migrations && yarn setup:development",
+    "install": "yarn install:schema && yarn install:semantic-ui && yarn install:ui && yarn install:backend && yarn install:common && yarn install:scripts && yarn install:migrations",
```

Comment:
how about parallelizing those? it might make installation faster, but the logs difficult to read